ui/src/app/modules/domain/column-response.ts

```diff
@@ -34,6 +34,42 @@ export function emptyColumnResponse(): ColumnResponse {
   };
 }
 
+export function deleteColumnResponse(
+  response: ResponseWithId,
+  items: ItemSorter
+): void {
+  function findIndex(
+    inputResponse: ResponseWithId,
+    responses: Array<Object>
+  ): number {
+    let index = -1;
+    for (let i = 0; i < responses.length; i++) {
+      const comparatorResponse: ResponseWithId = responses[i] as ResponseWithId;
+      if (comparatorResponse.id === inputResponse.id) {
+        index = i;
+        break;
+      }
+    }
+
+    return index;
+  }
```

Comment:
findIndex already exists on arrays. You could do
`items.active.findIndex((item: ResponseWithId) => item.id === response.id);`
`items.completed.findIndex((item: ResponseWithId) => item.id === response.id);`

Reply:
done